src/backend/distributed/shared_library_init.c

```diff
@@ -1006,6 +1012,17 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomEnumVariable(
+		"citus.coordinator_aggregation_strategy",
+		gettext_noop("Sets the strategy for when an aggregate cannot be pushed down."),
+		NULL,
+		&CoordinatorAggregationStrategy,
+		COORDINATOR_AGGREGATION_DISABLED,
```

Comment:
If row-gather works reliably then users will likely be happier if we make it the default.

Reply:
Bit worried that users will stumble into this when they try to do something like `sum(distinct)`, but updated default

Reply:
You're probably right, but would be preferable to use max_intermediate_result_size. That has so far proven to not be a major annoyance.